scrapers/Traxxx.yml

```diff
@@ -0,0 +1,49 @@
+name: "Traxxx"
+sceneByURL:
+  - url:
+     - traxxx.me/actor/
```

Comment:
it should be `scene` here... not actor
and you havent added  the `scene_url` case in the python file....

Reply:
nice catch ty